client/src/views/files/utils.ts

```diff
@@ -13,6 +13,10 @@ import { modalController, popoverController } from '@ionic/vue';
 import { Answer, askQuestion, I18n, MsModalResult } from 'megashark-lib';
 import { showSaveFilePicker } from 'native-file-system-adapter';
 
+export function getTmpFileName(fileName: string): string {
+  return `._${fileName}`;
```

Comment:
What if a file with this name already exists? It's easier to just use something like `crypto.randomUUID()` to avoid potential collisions. This way we never have to worry about them.